City-of-Helsinki/drupal-helfi-kymp
# UHF-2478: Genesys chat block placement

How to test:

- Set up KYMP
- Checkout this branch: `git checkout UHF-2478_genesys_chat_block_placement`
- Import config: `make drush-cim`
- Clear cache: `make drush-cr`
- See the block config and verify that
  - "Genesys Chat" block is in "Attachments" region below the "Smartti Chatbot" block
  - "Genesys Chat" block is configured as follows:
    - "Display title" is disabled
    - Language: Finnish & Content
